cdap-ui/app/cdap/components/Experiments/FieldLevelLineage/Table/index.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React from 'react';
+import './Table.css';
+
+interface INode {
+  id: string;
+  name: string;
+  group: number;
+}
+
+interface ITableProps {
+  nodes: INode[];
+  tableName: string;
+}
+
+export default function Table({ nodes, tableName }: ITableProps) {
```

Comment:
Based on our conversation earlier could you try out if our existing `SortableStickyGrid` works for this case? We don't need to reinvent the wheel. We can try to experiment with it in the next iteration. The very least can you add a comment about the `SortableStickyGrid` usage?